packages/twenty-front/src/pages/settings/data-model/SettingsObjectDetail.tsx

```diff
@@ -90,6 +91,9 @@ export const SettingsObjectDetail = () => {
 
   return (
     <SubMenuTopBarContainer Icon={IconSettings} title="Settings">
+      <PageTitle
```

Comment:
As said in my above comment in title-utils I don't think we want that granularity in those components